src/components/LandingPage/FAQ/FAQ.tsx

```diff
@@ -12,18 +12,33 @@ interface FAQDropDownProps {
 
 const FAQDropDown: React.FC<FAQDropDownProps> = (props) => {
   const { question, answer } = props
+  let [opened, setOpen] = useState(false)
+  let [mounted, setMounted] = useState(false)
+
+  useEffect(() => {
+    setMounted(true)
+  }, [])
+
+  const toggleOpen = () => {
+    setOpen(!opened)
+  }
   return (
-    <div className="w-[90vw] md:w-[60vw] collapse border border-base-300 bg-base-100 rounded-box">
-      <input type="checkbox" />
+    <a
+      className="w-[90vw] md:w-[60vw] collapse border border-base-300 bg-base-100 rounded-box"
```

Comment:
w-full should fix it

Reply:
w-full does not fix it, the faq's were too long compared to the rest of the page content to begin with, i recentered them instead